setup.cfg

```diff
@@ -0,0 +1,51 @@
+# Configuration file written in INI language.
+# Sort all entries alphabetically!
+
+# Configuration of flake8
+# See: https://flake8.pycqa.org/en/latest/user/configuration.html
+[flake8]
+count = True
+docstring-convention = numpy
+# doctests will be run by sphinx via "make -C ./docs/ doctest"
+doctests = False
```

Comment:
> Does sphinx linting?

No

> My understanding is that sphinx simply tries to run the examples from the docs (and checks the output if any reference is given).

Yes (or better: Not Sphinx itself runs the examples from the docs, but [doctest](https://docs.python.org/3/library/doctest.html) does this. Sphinx offers a convenient way to run doctest via Sphinx's Makefile with `make -C ./docs/ doctest`).

> This means that the code style of the examples is not checked, isn't it?

No, it is. The docs (and the examples therein) are still linted, but the examples are not tested with doctest by flake8 (or better PyFlakes). From the [docs of flake8](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-doctests): `doctests = True` "Enable[s] PyFlakes syntax checking of doctests in docstrings." If you set `doctests = True`, you get errors like `NameError: name 'mdt' is not defined`, because I do not import MDTools (and NumPy) in all examples explicitly. Rather I defined a global doctest setup in [conf.py](https://github.com/andthum/mdtools/blob/66217f76915efd79bd84930b06e5ded1bbb11fa8/docs/source/conf.py#L367), which imports these two packages in all examples. flake8 isn't aware of this and throws errors about unknown names. So in short, `doctests = False` only tells flake8 to not run doctest.

Reply:
Ok, that makes sense. I wasn't completely sure that both run the identical doctests command.